src/main/kotlin/racingcar/domain/racing/RacingCarAccelerator.kt

```diff
@@ -0,0 +1,19 @@
+package racingcar.domain.racing
+
+import racingcar.domain.car.CarAccelerator
+import racingcar.domain.car.CarAction
+
+class RacingCarAccelerator : CarAccelerator {
+
+    override fun trigger(): CarAction {
+        return if (MOVE_CONDITION.random() >= MOVE_CONDITION_STANDARD) CarAction.MOVE
+        else CarAction.STOP
```

Comment:
전체 표현식이 한 줄에 들어가는 경우에만 중괄호를 생략할 수 있습니다. 아래의 Kotlin style guide를 참고해 주세요.
https://developer.android.com/kotlin/style-guide#expressions

```suggestion
        return if (MOVE_CONDITION.random() >= MOVE_CONDITION_STANDARD) {
            CarAction.MOVE
        } else {
            CarAction.STOP
        }
```

Reply:
https://github.com/next-step/kotlin-racingcar/pull/238/commits/0cfe70b76aa1849e97a7ab1b172455558c498b93 새로 알게 되었네요. 감사합니다!